ci/azure-pipelines.yml

```diff
@@ -28,9 +28,20 @@ jobs:
         CODECOV_UPLOAD_TOKEN: $(CODECOV_UPLOAD_TOKEN)
       displayName: Upload coverage to Codecov
 
+  - job: BDDTest
+    pool:
+      vmImage: ubuntu-16.04
```

Comment:
Assume we can update this to the latest ubuntu LTS?

Reply:
Fixed